### PR TITLE
TEC-477 - Add subscribe link to main an mobile navigation

### PR DESCRIPTION
**Pull request by umbobabo**
https://jira.economist.com/browse/TEC-477

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -124,7 +124,8 @@
     "lint": "eslint $npm_package_config_lint_opts .",
     "pages": "git stash save -u pages-stash && git branch -D gh-pages; git checkout --orphan gh-pages && git add -f $npm_package_config_ghpages_files && git commit -anm'ghpages' && git push origin HEAD:gh-pages -f",
     "prepages": "npm run doc",
-    "prepublish": "babel . -d . -x .es6 --ignore node_modules",
+    "removebadassets": "rm assets/icons.svg",
+    "prepublish": "removebadassets && babel . -d . -x .es6 --ignore node_modules",
```
Comment (fabiosantoscode) [inline package.json]: Same question, isn't it `npm run removebadassets`?

**Comment by umbobabo** [comment]
You are right mate!
Corrected.

**Comment by umbobabo** [comment]
I reverted the command, anyway I'm trying to update to the latest version of our devpack that should fix this issue.

@fabiosantoscode pls review.

**Comment by fabiosantoscode** [comment]
:rainbow:

**Pull request merged**